projects/core/components/loader/loader.style.less

```diff
@@ -54,6 +48,33 @@
     &._loading {
         overflow: hidden;
     }
+
+    // Use native CSS max function instead LESS one
+    stroke-width: ~'max(calc(var(--t-size) / 12), 1.5px)';
```

Comment:
It is ok to use in v4.0
https://caniuse.com/css-math-functions

![Screenshot 2024-01-25 at 16 49 10](https://github.com/taiga-family/taiga-ui/assets/35179038/bd9dbab3-d729-4bfb-8e3c-63ba487d942b)